omx/gstomxaudiodec.c

```diff
@@ -664,15 +663,8 @@ gst_omx_audio_dec_loop (GstOMXAudioDec * self)
       gst_pad_pause_task (GST_AUDIO_DECODER_SRC_PAD (self));
       self->started = FALSE;
     } else if (flow_ret == GST_FLOW_FLUSHING) {
-      GST_DEBUG_OBJECT (self, "Flushing -- stopping task");
-      g_mutex_lock (&self->drain_lock);
-      if (self->draining) {
-        self->draining = FALSE;
-        g_cond_broadcast (&self->drain_cond);
-      }
-      gst_pad_pause_task (GST_AUDIO_DECODER_SRC_PAD (self));
+      GST_DEBUG_OBJECT (self, "Flushing ..");
```

Comment:
We could remove the two points, or at least add the third point.

Reply:
Got it. 66ebc2515adec0b1de7ca6ee7d60380af59d1fad